### PR TITLE
Bug 2071715: fix 404 on Environments nav

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://bugzilla.redhat.com/show_bug.cgi?id=2071715

**Solution description:**
Removed `namespaced: true` from the nav item's extension

**Screeshot:**
Screenshot from 2022-04-13 23-24-15

/kind bug

### CODE REVIEW

**Review by rottencandy — APPROVED** [review]
/lgtm

File: frontend/packages/gitops-plugin/console-extensions.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
       "insertBefore": "helm",
       "name": "%gitops-plugin~Environments%",
       "href": "/environments",
-      "namespaced": true,
```
Comment (invincibleJai) [inline frontend/packages/gitops-plugin/console-extensions.json:12]: is it not namespaced scoped?

and if it's based on gitops operator can't we have it feature flagged?

cc @divyanshiGupta

Comment (debsmita1) [reply frontend/packages/gitops-plugin/console-extensions.json:12]: @invincibleJai no, it shouldn't be namespace scoped, confirmed it with @reginapizza 
And, it is already feature flagged

Comment (invincibleJai) [reply frontend/packages/gitops-plugin/console-extensions.json:12]: Thanks for the update, missed seeing the feature flag with `OPENSHIFT_GITOPS`

**Pull request merged**